tests/e2e/containers/config.go

```diff
@@ -0,0 +1,77 @@
+package containers
+
+// ImageConfig contains all images and their respective tags
+// needed for running e2e tests.
+type ImageConfig struct {
+	InitRepository string
+	InitTag        string
+
+	JunoRepository string
+	JunoTag        string
+
+	RelayerRepository string
+	RelayerTag        string
+}
+
+//nolint:deadcode
+const (
+	// Current Git branch juno repo/version. It is meant to be built locally.
+	// It is used when skipping upgrade by setting JUNO_E2E_SKIP_UPGRADE to true).
+	// This image should be pre-built with `make docker-build-debug` either in CI or locally.
+	CurrentBranchRepository = "juno"
+	CurrentBranchTag        = "debug"
+	// Pre-upgrade juno repo/tag to pull.
+	// It should be uploaded to Docker Hub. JUNO_E2E_SKIP_UPGRADE should be unset
+	// for this functionality to be used.
+	previousVersionRepository = "ducnt87/juno"
```

Comment:
is this right?

Reply:
Notional team can't access to CosmosContract DockerHub, so we need to create one by personal account. I think better if we can automate creating this and uploading it by the official DockerHub account.

Reply:
Hi, This is a temporary solution for pulling image from my Dockerhub. I hope to be able to push it to an official Juno account in the future.

Reply:
And I wills setup a bot that automatically creates images on DockerHub every time a new version is available. To do this I will need an official Dockerhub of Juno.

Reply:
If you are in the notional team for Juno you should be able to make branches on this repo, then you can use the github images docker repository, if that's what you mean?

Reply:
Right!!! But I don't have permission to make a branch on Juno right now, I will ask Vuong to create it.

Reply:
@the-frey 
seem I need more permission for push to ghcr. It's my log :
```
The push refers to repository [[ghcr.io/cosmoscontracts/juno-e2e](http://ghcr.io/cosmoscontracts/juno-e2e?fbclid=IwAR1NXhkAZvYN6JgMM44mF41fQ-I-qJMGDP0_vllLNwTNUAQ9peAv1BMeFIw)]
8a2d6dd588ba: Preparing 
36a41810fd3a: Preparing 
34d5ebaa5410: Preparing 
denied: permission_denied: The token provided does not match expected scopes.
```

Reply:
@the-frey I have uploaded the image to ghcr.io/cosmoscontracts/juno...Can you review it? Many thanks!!